vital/types/track.cxx

```diff
@@ -45,29 +45,28 @@ namespace {
 class compare_state_frame
 {
 public:
-  bool operator()( const kwiver::vital::track_state_sptr& ts, kwiver::vital::frame_id_t frame )
+  bool operator()( const kwiver::vital::track_state_sptr& ts,
+                   kwiver::vital::frame_id_t frame )
   {
     return ts && ts->frame() < frame;
   }
 };
 
 }
 
-
 namespace kwiver {
-namespace vital {
 
+namespace vital {
 
-/// Default Constructor
+// ----------------------------------------------------------------------------
 track
 ::track(track_data_sptr d)
   : id_( invalid_track_id )
   , data_(d)
 {
 }
 
-
-/// Copy Constructor
```

Comment:
Just for my own benefit, is this the new KWIVER style to not provide any documentation in the cxx file?  Long ago (in MAP-Tk days) we decided to duplicate just the single brief documentation line from the header into the cxx to provide some context to developers.  Have decided to drop that doc in the cxx going forward?

Reply:
I was not aware of that... it was more my [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) reflex (and also wanting to reformat with consistent separator lines). That said, "copy constructor" is redundant to the point of being anti-helpful. The others range from equally bad to arguable.

I could entertain an argument for that style if doing so adds *useful* information to the `.cxx`. I don't feel that was the case here, whereas the duplication has an obvious maintenance cost.